### DISCUSSION TITLE
What are the Disadvantages of Using a Public Runtime Hosted in the Bref AWS Account?

### DISCUSSION THREAD

**Discussion by jericopulvera** [brefphp/bref#1387] [Support]
If the runtime hosted in the Bref AWS account is deleted, will this result in downtime for a production application or will it not affect the deployed application at all?

**Comment by mnapoli** [comment]
Someone once asked AWS support about that. I can't retrieve the link but the response was basically: existing Lambda functions will not stop working, however deployments will not work anymore.
